lib/matplotlib/axis.py

```diff
@@ -40,6 +40,10 @@
 }
 
 
+def new_func(x):
```

Comment:
This test func should be removed, it did error on CI.

Reply:
👍 I removed the commit with this in